# Goal definition
float32 x_min
float32 x_max
float32 y_min
float32 y_max
---

---
#Feedback definition
geometry_msgs/Pose current_pose
